build.sbt

```diff
@@ -64,6 +78,8 @@ libraryDependencies ++= {
 // https://github.com/scala/pickling/issues/10
 scalacOptions ++= Seq("-deprecation", "-unchecked", "-feature", "-Xmax-classfile-name", "200")
 
+addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full)
```

Comment:
I'm curious why macro-paradise is needed -- is it a requirement for circe-generic?